.github/workflows/detect-breaking-changes-report.yml

```diff
@@ -12,6 +12,11 @@ jobs:
     env:
       ARTIFACT_NAME: 'levitate' # The name of the artifact that we would like to download
       ARTIFACT_FOLDER: '${{ github.workspace }}/tmp' # The name of the folder where we will download the artifact to
+    permissions:
+      contents: write
+      issues: write
+      pull-requests: write
+      artifacts: read
```

Comment:
Hmm, why do we actually need these explicit permissions? (I mean in what scenarios were the current permissions of the workflow not sufficient?)

Reply:
Organizations with restrictive default permissions as noted in the issue.

Here's an example:
https://github.com/check-spelling/grafana/actions/runs/4130829125/jobs/7137892469#step:1:17

Reply:
Fair enough 👍